### PR TITLE
feat: add EvaluationStorage

### PR DESCRIPTION
**Pull request by yshrsmz**
add base LocalStorage implementation and Storage implementation for Evaluations

### CODE REVIEW

File: src/internal/evaluation/EvaluationStorage.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+export interface EvaluationStorage {
+  add( evaluations: Evaluation[]): void
+  get(): Record<string, Evaluation>
+  getByFeatureId(featureId: string): Evaluation | null
+  deleteAllAndInsert(evaluations: Evaluation[]): void
+}
+
+export class EvaluationStorageImpl implements EvaluationStorage {
```
Comment (yshrsmz) [inline src/internal/evaluation/EvaluationStorage.ts:16]: I'm planning to implement EvalutinInteractor as I did in Android SDK, but not going to add an in-memory cache and use this EvaluationStorage directly.

LocalStorage should be fast enough.

Comment (cre8ivejp) [reply src/internal/evaluation/EvaluationStorage.ts:16]: Sounds good.
Do you know if the current SDK is using an in-memory cache?

**Review by cre8ivejp — APPROVED** [review]
Thank you!

File: src/internal/evaluation/EvaluationStorage.ts (hunk 1 above)
Comment (yshrsmz) [reply src/internal/evaluation/EvaluationStorage.ts:16]: Yes, it's using in-memory cache.
https://github.com/ca-dp/bucketeer-web-sdk/blob/master/src/storage/storage.ts#L31

**Pull request merged**